examples/release_highlights/plot_release_highlights_0_24_0.py

```diff
@@ -0,0 +1,236 @@
+                     LogisticRegression(max_iter=1000))
+X_train, X_test, y_train, y_test = train_test_split(X, y, train_size=5000,
+                                                    test_size=10000,
+                                                    random_state=42)
+pipe.fit(X_train, y_train).score(X_test, y_test)
+
+linear_baseline = make_pipeline(MinMaxScaler(),
+                                LogisticRegression(max_iter=1000))
+linear_baseline.fit(X_train, y_train).score(X_test, y_test)
+
+##############################################################################
+# Individual Conditional Expectation
+# ----------------------------------
+# A new kind of partial dependence plot is available: the Individual
+# Conditional Expectation (ICE) plot. It visualizes the dependence of the
+# prediction on a feature for each sample separately with one line per sample.
+# See the :ref:`User Guide <individual_conditional>`
+
+from sklearn.linear_model import BayesianRidge
+from sklearn.datasets import fetch_california_housing
+from sklearn.inspection import plot_partial_dependence
+
+X, y = fetch_california_housing(return_X_y=True, as_frame=True)
+features = ['MedInc', 'AveOccup', 'HouseAge', 'AveRooms']
+est = BayesianRidge()
+est.fit(X, y)
+display = plot_partial_dependence(
+       est, X, features, kind="individual", subsample=50,
+       n_jobs=3, grid_resolution=20, random_state=0
+)
+display.figure_.suptitle(
+    'Partial dependence of house value on non-location features\n'
+    'for the California housing dataset, with BayesianRidge'
+)
+display.figure_.subplots_adjust(hspace=0.3)
+
+##############################################################################
+# DecisionTreeRegressor now supports the new 'poisson' splitting criterion 
+# ------------------------------------------------------------------------
+# The integration of Poisson regression estimation continues from version 0.23
+# :class:`~sklearn.tree.DecisionTreeRegressor` now supports a new `'poisson'`
+# splitting criterion. Setting `criterion="poisson"` might be a good choice
+# if your target is a count or a frequency.
+
+from sklearn.tree import DecisionTreeRegressor
+
+n_samples, n_features = 1000, 20
+rng = np.random.RandomState(0)
+X = rng.randn(n_samples, n_features)
+# positive integer target correlated with X[:, 5] with many zeros:
+y = rng.poisson(lam=np.exp(X[:, 5]) / 2)
+X_train, X_test, y_train, y_test = train_test_split(X, y, random_state=rng)
+regressor = DecisionTreeRegressor(criterion='poisson', random_state=0)
+regressor.fit(X_train, y_train)
+
+##############################################################################
+# New metrics available
+# ---------------------
+# A number of new metric functions are now available, for example
+# :func:`metrics.top_k_accuracy_score` and :func:`metrics.det_curve`.
+# For a complete list see the `changelog
+# <../../whats_new/v0.24.html#sklearn-metrics>`_.
```

Comment:
For more discoverability and avoid an additional click, shouldn't we mention all of them here instead?

Reply:
To be honest I find a six element bullet list quite long: it will be just a duplicate of the list in the Changelog.

Reply:
It would just make this section as long as the other ones, wouldn't it?  For now it's just 2 lines (and one plot)

It's not immediately obvious in the changelog which ones are new metrics and which ones are not, as entries are mixed. I think that those that we don't list here will simply not be discovered via the highlights.

Reply:
There are only 3 new metrics: `top_k_accuracy_score`, `mean_absolute_percentage_error`, and `rand_score`.

`det_curve`, currently mentioned, isn't a new metric.

Since I'm being invited to address my own comments (https://github.com/scikit-learn/scikit-learn/pull/18795#issuecomment-735675493) and since 3 new unrelated metrics aren't worth a highlight IMHO, I'm going with my initial gut feeling (https://github.com/scikit-learn/scikit-learn/pull/18795#discussion_r528174950) which is to remove this section.

Feel free to put it back though.